tests/scripts/setup

```diff
@@ -21,9 +21,16 @@ def create_node(opts)
 end
 
 def run_puppet_on_nodes
-  docker_hosts.each do |name, ip|
-    #Use a bogus tag to only run pluginsync
-    `docker exec -d #{name} puppet agent -t --tags bogus`
+  manifests_dir = '/etc/puppetlabs/code/environments/production/manifests/'
+  begin
+    File.rename(File.join(manifests_dir, 'site.pp'), File.join(manifests_dir, 'site.pp.bak'))
```

Comment:
This bit is slightly fragile (eg, it seems like it might be subject to timing issues or race conditions.) I like that you put the corresponding rename back call in an `ensure` block. Could you also tag this with a `# TODO` comment to remind you to come back and find a cleaner way someday?

You'll also want a note explaining why you're doing this weird thing. Future maintainers of the code will appreciate knowing what it's intended to accomplish.

Reply:
@binford2k That's a good point. I've added a comment, but I'll also do a little bit more testing to see if I can force a timing issue here. The docker exec here doesn't wait for completion before moving on (hence the polling later on), so it's plausible that the script would move on and swap the site.pp back before the compiler reads it.